### ISSUE TITLE
loss of vae

### ISSUE THREAD

**Issue by henbucuoshanghai** [hpcaitech/Open-Sora#651]
self.logvar = nn.Parameter(torch.ones(size=()) * logvar_init)
 nll_loss = recon_loss / torch.exp(self.logvar) + self.logvar

why using self.logvar?it is trainable?

**Comment by henbucuoshanghai** [comment]
# == loss: image only distillation ==
                if cfg.get("use_image_identity_loss", False) and x.size(2) == 1:
                    _, image_identity_loss, _ = vae_loss_fn(x_z, z,


distillation ？why using vae_loss_fn(x_z, z,）？why    (x_z, z,）？
